cpp/include/raft/spatial/knn/detail/brute_force_knn.hpp

```diff
@@ -0,0 +1,342 @@
+#pragma once
+
+#include <raft/cudart_utils.h>
+#include <raft/cuda_utils.cuh>
+
+#include <faiss/gpu/GpuDistance.h>
+#include <faiss/gpu/GpuResources.h>
+#include <faiss/gpu/StandardGpuResources.h>
+#include <faiss/utils/Heap.h>
+#include <faiss/gpu/utils/Limits.cuh>
+#include <faiss/gpu/utils/Select.cuh>
+
+#include <raft/linalg/distance_type.h>
+#include <thrust/device_vector.h>
+#include <thrust/iterator/transform_iterator.h>
+#include <iostream>
+#include <raft/handle.hpp>
+#include <set>
+
+#include "processing.hpp"
+
+namespace raft {
+namespace knn {
```

Comment:
We should probably also include the `spatial` namespace in the hierarchy here for consistency.

Reply:
Fixed here https://github.com/rapidsai/raft/pull/126/commits/9315752555050877f934522a08709b5dc2daff8a